### ISSUE TITLE
Using prometheus as metrics provider

### ISSUE THREAD

**Issue by Jack-Kingdom** [kubernetes/dashboard#8665]
### What would you like to be added?

support using prometheus metrics info as metrics source.

### Why is this needed?

metrics-scraper store metrics info for provide, and prometheus did this job too. We have no need store those data twice.

**Comment by avanish23** [comment]
Hi @Jack-Kingdom 
The ask seems to be able to disable the default metrics scraper that comes with the dashboard as you already have Prometheus collecting them for you. 
Is my understanding right?

**Comment by Jack-Kingdom** [comment]
Yeah, I mean, we can disable the default metrics scraper and use prometheus as metrics source.

**Comment by floreks** [comment]
If I understand correctly you would want dashboard to support prometheus as data source and display its' metrics directly in the UI?

**Comment by Jack-Kingdom** [comment]
@floreks Right!

I found we use sqlite as storage in metrics scraper. Use relation-based db as time seriesed db is not a good idea, this may cause performance issues in large scale k8s.

we can make metrics scraper as a adapter layer to support different metrics source. In my mind, most of k8s deployment have their own metrics system.

**Comment by floreks** [comment]
Supporting other data sources is on our list for sure. Victoria metrics/prometheus would be first.

Metrics scraper can't really be compared to them since it is designed to scrape only basic metrics and for a very short time. Normally it only stores the last 15 minutes to visualize current usage and short time peaks. It also scrapes only from metrics server, where i.e. prometheus has its own node-level scraper. For what it is being used is totally fine.

**Comment by valyala** [comment]
How about using [vmagent](https://docs.victoriametrics.com/victoriametrics/vmagent/) as a lightweight metrics' scraper, which could send the scraped metrics to any external database, which supports receiving metrics via Prometheus remote_write protocol? Both Prometheus and VictoriaMetrics supports receiving metrics via this protocol.
